common/remote/sockfile.cpp

```diff
@@ -6355,14 +6357,15 @@ class CRemoteFileServer : implements IRemoteFileServer, public CInterface
                             }
                             else
                             {
-                                responseWriter->outputBeginNested("Eog", false);
-                                responseWriter->outputEndNested("Eog");
                                 row = outputActivity->nextRow(outBuilder, rowSz);
                                 if (!row)
                                 {
                                     eoi = true;
                                     break;
                                 }
+                                responseWriter->outputEndArray("Row");
+                                responseWriter->outputBool(true, "Eog");
```

Comment:
This Eog has to be inside a new message object, so:
responseWriter->outputEndArray("Row");
responseWriter->outputBeginNested("Msg");
responseWriter->outputBool(true, "Eog");
responseWriter->outputEndNested("Msg");
responseWriter->outputBeginArray("Row");

Reply:
I think needs to be.
                                responseWriter->outputEndArray("Row");
                                responseWriter->outputEndNested("Msg");
                                responseWriter->outputBeginNested("Msg", false);
                                responseWriter->outputBool(true, "Eog");
                                responseWriter->outputEndNested("Msg");
                                responseWriter->outputBeginNested("Msg", false);
                                responseWriter->outputBeginArray("Row");
but please check.